wandb/apis/public/artifacts.py

```diff
@@ -61,11 +96,15 @@ def __init__(
 
     @property
     def length(self) -> None:
+        """Returns `None`."""
         # TODO
         return None
 
     @property
     def more(self):
+        """Returns `True` if there are more artifacts to fetch. Returns
+        `False` if there are no more files to fetch.
```

Comment:
By Python convention the first line has to fit on one 80-character line (sometimes 88), including the indentation and the triple quote.

In this case, the fix is pretty simple:

```python
"""Returns whether there are more artifacts to fetch."""
```